src/librustdoc/clean/utils.rs

```diff
@@ -453,3 +453,19 @@ crate fn find_nearest_parent_module(tcx: TyCtxt<'_>, def_id: DefId) -> Option<De
         None
     }
 }
+
+/// Checks for the existence of a `doc(foo)` attribute. For example,
+/// checks for the existence of `spotlight` in the attribute below:
+///
+/// ```text
+/// #[doc(spotlight)]
+/// ```
```

Comment:
It would be nice to make this a valid doc-test.
```suggestion
/// checks for the existence of `hidden` in the attribute below:
///
/// ```
/// #[doc(hidden)]
/// pub fn foo() {}
/// ```
```